bundle_pt.properties

```
@@ -248,8 +248,8 @@ level=Nível
 
 # Moderate
 
-slap=Tapa
-kick=Chutar
+slap=Aviso
+kick=Kickar
```

The former makes more sense, we're localizing it not making up words
```suggestion
kick=Expulsar
```